src/EditorFeatures/CSharpTest/AddObsoleteAttribute/AddObsoleteAttributeTests.cs

```diff
@@ -94,7 +94,7 @@ class Derived : [||]Base {
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddObsoleteAttribute)]
         public async Task TestObsoleteClassUsedInField()
         {
-            await TestInRegularAndScript1Async(
```

Comment:
❓ Can you undo this method rename and just update the implementation of that method to use C# 7.3? IDE team will go through and update the implementation to remove that restriction separately.

Reply:
I'm also right now doing a pass to simply disable/fix the tests as appropriate.

Reply:
Sure, I can do the rename back, which should reduce the code churn

Reply:
@sharwell I take it back, I can't do that. Some tests still call the old version, so a rename would mean those tests break because they now use C# 7.3